azkaban-db/src/main/sql/create.execution_flows.sql

```diff
@@ -12,6 +12,7 @@ CREATE TABLE execution_flows (
   enc_type    TINYINT,
   flow_data   LONGBLOB,
   executor_id INT                   DEFAULT NULL,
+  use_executor INT,
```

Comment:
do we need to set the default value here?

Reply:
no need to set it explicitly, default is NULL

Reply:
it's still better to set it explicitly 1. different DBMS might have different rules 2. being consistent with executor_id statement as above 3. easy for reader to know the default value of this column.

Reply:
ok, but we might want to specify it for the rest of the columns as well.